src/js/modules/core/components/PageLayout.js

```diff
@@ -6,6 +6,7 @@ import injectSheet from "react-jss";
 import PageHeader from "./PageHeader";
 import PageFooter from "./PageFooter";
 import { getTopLevelSectionsWithDirectChildren } from "../../sections/selectors";
+import { getAboutPages} from "../../about/selectors";
```

Comment:
Rename to `getAbouts`. It's not the best name, but this way we keep things consistent.